plugins/inventory/aws_ec2.py

```diff
@@ -73,8 +73,8 @@
     description:
       - A list of filters. Any instances matching at least one of the filters are included in the result.
       - Available filters are listed here U(http://docs.aws.amazon.com/cli/latest/reference/ec2/describe-instances.html#options).
-      - Every entry in this list triggers a search query. As such, from a performance point of view, it's better to
-        keep the list as short as possible.
+      - Every entry in this list triggers a search query if C(apply_and_logic_on_include_filters) is not set to I(true).
```

Comment:
I'm not sure though, but shouldn't this be the default behaviour instead of defining a new parameter?
@tremble

Reply:
setting this as default behavior will lead to a breaking change knowing that current behavior is also okay, why then modify it?